app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Could you move the following into an `Auth` namespace please?
- `ConfirmationsController`
- `OmniauthCallbacksController`
- `RegistrationsController`
- `SessionsController`